types/ckeditor4/index.d.ts

```diff
@@ -1,2755 +1,58 @@
-// Type definitions for CKEditor 4.16
+// Type definitions for CKEDITOR 4.20
 // Project: https://ckeditor.com/ckeditor-4/, http://ckeditor.com
 // Definitions by: Thomas Wittwer <https://github.com/wittwert>
 //                 Stuart Long <https://github.com/stuartlong>
+//                 Viktor Pegy <https://github.com/viktorpegy>
```

Comment:
Please remove this owner per https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63591#issuecomment-1466425870

Reply:
Removed